server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcResolver.java

```diff
@@ -129,110 +162,76 @@
 
-                    LOG.debug("OneField content (conversion from TEXT): '{}'", rawVal);
-                    switch (DataType.get(column.columnTypeCode())) {
-                        case VARCHAR:
-                        case BPCHAR:
-                        case TEXT:
-                        case BYTEA:
-                            break;
-                        case BOOLEAN:
-                            oneField.val = Boolean.parseBoolean(rawVal);
-                            break;
-                        case INTEGER:
-                            oneField.val = Integer.parseInt(rawVal);
-                            break;
-                        case FLOAT8:
-                            oneField.val = Double.parseDouble(rawVal);
-                            break;
-                        case REAL:
-                            oneField.val = Float.parseFloat(rawVal);
-                            break;
-                        case BIGINT:
-                            oneField.val = Long.parseLong(rawVal);
-                            break;
-                        case SMALLINT:
-                            oneField.val = Short.parseShort(rawVal);
-                            break;
-                        case NUMERIC:
-                            oneField.val = new BigDecimal(rawVal);
-                            break;
-                        case TIMESTAMP:
-                            boolean isConversionSuccessful = false;
-                            for (SimpleDateFormat sdf : timestampSDFs.get()) {
-                                try {
-                                    java.util.Date parsedTimestamp = sdf.parse(rawVal);
-                                    oneField.val = new Timestamp(parsedTimestamp.getTime());
-                                    isConversionSuccessful = true;
-                                    break;
-                                } catch (ParseException e) {
-                                    // pass
-                                }
-                            }
-                            if (!isConversionSuccessful) {
-                                throw new ParseException(rawVal, 0);
-                            }
-                            break;
-                        case DATE:
-                            oneField.val = new Date(dateSDF.get().parse(rawVal).getTime());
-                            break;
-                        default:
-                            throw new UnsupportedOperationException(
-                                    String.format("Field type '%s' (column '%s') is not supported",
-                                            DataType.get(oneField.type),
-                                            column));
-                    }
+                if (oneField.val == null) {
+                    continue;
                 }
-            }
 
-            column_index += 1;
+                String rawVal = (String) oneField.val;

```

Comment:
this log might still be useful for debugging purposes.
```suggestion
                String rawVal = (String) oneField.val;
                LOG.debug("OneField content (conversion from TEXT): '{}'", rawVal);
```

Reply:
Thanks for this comment. I have changed the logging logic so that it includes output for fields of all types. This increases the size of log file, but can be disabled without rebuilding PXF.